spec/spec_helper.rb

```diff
@@ -6,7 +6,10 @@
 require 'webmock/rspec'
 SimpleCov::Formatter::LcovFormatter.config.report_with_single_file = true
 SimpleCov.formatter = SimpleCov::Formatter::LcovFormatter
-SimpleCov.start
+SimpleCov.start do
+  track_files 'app/**/*.rb'
+  add_filter '/spec/'
+end
```

Comment:
It seems that SimpleCov has pre-defined configurations for Rails: https://github.com/simplecov-ruby/simplecov?tab=readme-ov-file#use-it-with-any-framework

Reply:
Ah that's convenient!